src/cpsr.rs

```diff
@@ -1,51 +1,293 @@
 use crate::condition::Condition;
+use crate::condition::ModeBits;
 
-/// Current Program Status Register.
+/// Current Program Status Register
```

Comment:
Keep the `.`
In the documentation is important to respect the grammar.

Reply:
:)